docs/spec/openapi.yml

```diff
@@ -700,6 +700,107 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+
+  '/namespaces/{namespace}/database-clusters/{dbName}/data-import-jobs':
+    x-everest-resource-name: data-import-jobs
+    get:
+      tags:
+      - Database Cluster
+      summary: List data import jobs for a database cluster
+      description: |
+        This API lists all data import jobs for the database cluster specified by the `name` and `namespace`.
+      operationId: listDataImportJobs
+      parameters:
+        - name: namespace
+          in: path
+          description: Name of the namespace
+          required: true
+          schema:
+            type: string
+        - name: dbName
+          in: path
+          description: Name of the database cluster. Can be found under Metadata["name"] of the DatabaseCluster object.
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DataImportJobList'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  '/namespaces/{namespace}/database-clusters/{dbName}/secret':
+    post:
+      tags:
+        - Database Cluster
+      summary: Create a secret for the given database cluster
+      description: |
+        This API creates a secret for the database cluster specified by the `name` and `namespace`
+      operationId: createDatabaseClusterSecret
+      parameters:
+        - name: dbName
```

Comment:
Do I understand properly that this secret with credentials shall be created before we create DBCluster? if yes - how could we know `dbName` beforehand?

Reply:
> Do I understand properly that this secret with credentials shall be created before we create DBCluster?

Not really, only the wizard step appears before the DB creation step. The API is called at the end of the wizard once the DBName is known